exporters/memory/test/in_memory_span_data_test.cc

```diff
@@ -0,0 +1,33 @@
+#include "opentelemetry/exporters/memory/in_memory_span_data.h"
+#include "opentelemetry/nostd/span.h"
+#include "opentelemetry/sdk/trace/span_data.h"
+
+#include <gtest/gtest.h>
+
+using opentelemetry::exporter::memory::InMemorySpanData;
+using opentelemetry::sdk::trace::Recordable;
+using opentelemetry::sdk::trace::SpanData;
+
+TEST(InMemorySpanData, AddRecordable)
+{
+  InMemorySpanData data(100);
+
+  ASSERT_EQ(0, data.GetSpans().size());
+
+  opentelemetry::nostd::span<std::unique_ptr<Recordable>> recordables(
```

Comment:
I think you can just define this as a span of SpanData instead of Recordable, since you just convert it back to SpanData later.

Reply:
Updated